manuscript/draft.Rmd

```diff
@@ -78,25 +78,31 @@ these approaches improve ecological forecasting in the future.
 Bird species richness was obtained from the Breeding Bird Survey (BBS)
 [@pardieck2017] using the Data Retriever software [@morris2013] and
 rdataretriever R package [@mcglinn2017]. The BBS data was filtered to exclude
-all nocturnal, cepuscular, and aquatic species
-[since these species are not well sampled by BBS methods; @hurlbert2005], as
-well as unidentified species, and hybrids. All data from routes that did not
-meet BBS acceptable criteria were excluded. Only routes within the continental
-United States were used so as to match the climate dataset. The resulting
-dataset included [[N]] species and [[N]] sites. Among sites richness varied from
-[[N]] to [[N]] with an average richness of [[N]] species. Within-site richness
-variation (among years) was small (~5 species).
+all nocturnal, cepuscular, and aquatic species [since these species are not well
+sampled by BBS methods; @hurlbert2005], as well as unidentified species, and
+hybrids. Due to limitations of the environmental data (below), we did not
+include any data from outside the continental United States. Observed richness
+from 1982 (the first complete year of NDVI data) to 2003 were used for training
+the models, and data from 2004 to 2013 was used for testing their performance.
+Only BBS routes that were sampled during 70% of the years in the training period
+were used (i.e., routes with at least 16 annual observations). All data from
+routes that did not meet BBS acceptable criteria were excluded, as were routes
+that had not been visited during at least 70% of the years in our model-training
+window. The resulting dataset included [[N]] species and [[N]] sites. Among
+sites,  richness varied from [[N]] to [[N]] with an average richness of [[N]]
+species. Within-site richness variation (among years) was small (standard
+deviation of about 5 species, after controlling for observer effects).
```

Comment:
I think I prefer the original separation of the discussion of the data itself and then later the decisions we made with respect to modeling it. This feels a bit like two things alternatingly mixed together. However I don't feel that strongly about it.

Either way I think we should cut the last sentence here and move it to the results if it isn't there already.